common/src/service/HttpClientHelper.cs

```diff
@@ -919,5 +894,37 @@ public void Dispose()
             Task<Exception> exception = mapToExceptionFunc(response);
             return await exception.ConfigureAwait(false);
         }
+
+        internal static HttpClient CreateDefaultClient(IWebProxy webProxy, Uri baseUri, int connectionLeaseTimeoutMilliseconds)
+        {
+
+#pragma warning disable CA2000 // Dispose objects before losing scope
+#if NETCOREAPP
+            SocketsHttpHandler httpMessageHandler = new SocketsHttpHandler();
```

Comment:
it looks like this is available only netcore2.1+, we should make this directive more specific

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.net.http.socketshttphandler?view=net-5.0#applies-to

Reply:
Good point. The code will look so ugly :(

Reply:
It already is, haha